frontend/src/pages/pipelines/global/experiments/BulkRestoreExperimentModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ListItem, Stack, StackItem } from '@patternfly/react-core';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
+import { BulkActionExpandableSection } from '~/pages/projects/components/BulkActionExpandableSection';
+import { BulkRestoreModal } from '~/concepts/pipelines/content/BulkRestoreModal';
+import { ExperimentListTabTitle } from '~/pages/pipelines/global/experiments/const';
+import { PipelineRunTabTitle } from '~/pages/pipelines/global/runs';
+
+interface BulkRestoreExperimentModalProps {
+  isOpen: boolean;
+  experiments: ExperimentKFv2[];
+  onCancel: () => void;
+}
+
+export const BulkRestoreExperimentModal: React.FC<BulkRestoreExperimentModalProps> = ({
+  isOpen,
+  experiments,
+  onCancel,
+}) => {
+  const { api } = usePipelinesAPI();
+  const onSubmit = React.useCallback(
+    () =>
+      Promise.all(
+        experiments.map((experiment) => api.unarchiveExperiment({}, experiment.experiment_id)),
+      ),
+    [api, experiments],
+  );
+  return (
+    <BulkRestoreModal
+      onCancel={onCancel}
+      isOpen={isOpen}
+      onSubmit={onSubmit}
+      resourceType="runs"
+      testId="bulk-restore-experiment-modal"
+    >
+      <Stack hasGutter>
+        <StackItem>
+          <b>{experiments.length}</b> experiments will be restored and returned to the{' '}
+          <b>{ExperimentListTabTitle.ACTIVE}</b> tab. Their runs and schedules can be restored from
+          their respective <b>{PipelineRunTabTitle.Archived}</b> and{' '}
```

Comment:
I don't think this statement is entirely true, at least right now, all archived runs exist in the "Archived" runs tab. They wouldn't go to the "Schedules" tab to restore anything.

Reply:
Actually, on top of this, Schedules are not archive-able to begin with, so just saying `Their runs can be restored from the "Archived" runs page` would be more appropriate IMO